awx/ui/src/screens/Inventory/shared/ConstructedInventoryHint.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import { t } from '@lingui/macro';
+import {
+  Alert,
+  AlertActionLink,
+  CodeBlock,
+  CodeBlockAction,
+  CodeBlockCode,
+  ClipboardCopyButton,
+} from '@patternfly/react-core';
+import {
+  TableComposable,
+  Thead,
+  Tr,
+  Th,
+  Tbody,
+  Td,
+} from '@patternfly/react-table';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+function ConstructedInventoryHint() {
+  const [copied, setCopied] = React.useState(false);
+
+  const clipboardCopyFunc = (event, text) => {
+    navigator.clipboard.writeText(text.toString());
+  };
+
+  const onClick = (event, text) => {
+    clipboardCopyFunc(event, text);
+    setCopied(true);
+  };
+
+  const pluginSample = `plugin: constructed
+strict: true
+use_vars_plugins: true
+groups:
+  shutdown: resolved_state == "shutdown"
+  shutdown_in_product_dev: resolved_state == "shutdown" and account_alias == "product_dev"
+compose:
+  resolved_state: state | default("running")`;
+
+  return (
+    <Alert
+      isExpandable
+      isInline
+      variant="info"
+      title={t`How to use constructed inventory plugin`}
+      actionLinks={
+        <AlertActionLink
+          href="https://docs.ansible.com/ansible/latest/collections/ansible/builtin/constructed_inventory.html"
```

Comment:
We should use this function https://github.com/ansible/awx/blob/devel/awx/ui/src/util/getDocsBaseUrl.js to help ensure that the user is taken to the correct version of the docs.